client/components/relatedWidget/Outfit.jsx

```diff
@@ -58,16 +58,16 @@ class Outfit extends React.Component {
   }
 
   componentDidMount() {
-    console.log(this.props.product)
+    // console.log(this.props.product);
     this.getImage(this.props.product.id);
     this.buildCarousel();
   }
 
   componentDidUpdate(prevProps) {
-    // if (this.props.yourOutfit !== prevProps.yourOutfit) {
+    if (this.props.yourOutfit !== prevProps.yourOutfit) {
       this.getImage(this.props.product.id);
       this.buildCarousel();
-    // }
+    }
   }
 
```

Comment:
If you are modifying other components to get some functionality working locally but don't want to be merging these changes in to other peoples code there are ways to avoid adding those files to commits. Lemme know if you would like me to share the system I've found that works well for me

Reply:
Yeah sorry about this mess in my files...this is what the master branch should look like though so hopefully there are no issues